src/Dialogs/Weather/RASPDialog.cpp

```diff
@@ -154,7 +191,12 @@ class RaspProviderRenderer : public ListItemRenderer {
 
   void OnPaintItem(Canvas &canvas, const PixelRect rc,
                    unsigned i) noexcept override {
-    row_renderer.DrawTextRow(canvas, rc, rasp_providers[i].name);
+    std::string str = items.at(i).name;
+    TCHAR *name=new TCHAR[str.size()+1];
```

Comment:
And now copying into a second heap allocation? Who is responsible for freeing this allocation? Does anybody? Is this a memory leak?